packages/jetpack-blocks/babel.config.js

```diff
@@ -0,0 +1,89 @@
+/** @format */
+const _ = require( 'lodash' );
+
+const targets = { browsers: [ 'last 2 versions', 'Safari >= 10', 'iOS >= 10', 'ie >= 11' ] };
+
+const config = {
+	presets: [
+		[
+			'@babel/env',
+			{
+				targets,
+				useBuiltIns: 'entry',
+				// allows es7 features like Promise.prototype.finally
+				shippedProposals: true,
+				// Exclude transforms that make all code slower, see https://github.com/facebook/create-react-app/pull/5278
+				exclude: [ 'transform-typeof-symbol' ],
+			},
+		],
+		'@babel/react',
+	],
+	plugins: _.compact( [
+		'@babel/plugin-proposal-class-properties',
+		'@babel/plugin-proposal-export-default-from',
+		'@babel/plugin-proposal-export-namespace-from',
+		'@babel/plugin-syntax-dynamic-import',
+		[
+			'@babel/transform-runtime',
+			{
+				corejs: false, // we polyfill so we don't need core-js
+				helpers: true,
+				regenerator: false,
+				useESModules: false,
+			},
+		],
+	] ),
+	overrides: [
+		{
+			test: [ './client/gutenberg/extensions', './packages/jetpack-blocks' ],
+			plugins: [
+				[
+					'@wordpress/import-jsx-pragma',
+					{
+						scopeVariable: 'createElement',
+						source: '@wordpress/element',
+						isDefault: false,
+					},
+				],
+				[
+					'@babel/transform-react-jsx',
+					{
+						pragma: 'createElement',
+					},
+				],
+			],
+		},
+	],
+	env: {
+		build_pot: {
+			plugins: [
+				[
+					'@wordpress/babel-plugin-makepot',
+					{
+						output: 'build/i18n-calypso/gutenberg-strings.pot',
+						headers: {
+							'content-type': 'text/plain; charset=UTF-8',
+							'x-generator': 'calypso',
+						},
+					},
+				],
+				[
+					'@automattic/babel-plugin-i18n-calypso',
+					{
+						dir: 'build/i18n-calypso/',
+						headers: {
+							'content-type': 'text/plain; charset=UTF-8',
+							'x-generator': 'calypso',
+						},
+					},
+				],
+			],
+		},
```

Comment:
I think we can drop the `build_pot` env here since we're not using i18n tools from within here.

Reply:
https://github.com/Automattic/wp-calypso/pull/31503/commits/384ab7421e1b54d4188ca187ce6a58ea83cb5420